libcodechecker/logger.py

```diff
@@ -106,12 +105,18 @@ def get_logger(name):
     return logging.getLogger(name)
 
 
+def upper(text):
+    """Convert text to uppercase """
+    return text.upper()
+
+
 def validate_loglvl(log_level):
     """
     Should return a valid log level name
     """
     log_level = log_level.upper()
-    if log_level not in map(string.upper, CMDLINE_LOG_LEVELS):
+
+    if log_level not in map(upper, CMDLINE_LOG_LEVELS):
```

Comment:
Use list comprehension instead of map(). This way there is no need to define a new function.